server/server.js

```diff
@@ -46,7 +47,9 @@ app.use("/api/admin/products", products);
 app.use("/api/user/address/get", address);
 app.use("/api/user/address/", address);
 app.use("/api/user/new/address/", address);
-app.use("/api/user/checkout", checkout);
+app.use("/api/user/razor", razor);
```

Comment:
don't use razor anywhere because payment gateways can change,
today we are using razor pay and tomorrow we might change to stripe or paypal

Reply:
I have changed